lib/miq_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class MiqDecorator
+  def initialize(instance)
+    @_decorated_instance = instance
+  end
+
+  def method_missing(*args)
+    @_decorated_instance.send(*args)
+  end
+end
```

Comment:
I would strongly suggest using an actual delegator rather than crossing your fingers with method_missing.

Are you familiar with some of Ruby's core Delegator classes? I would use [`SimpleDelegator`](http://ruby-doc.org/stdlib-2.4.0/libdoc/delegate/rdoc/SimpleDelegator.html) here.

`class MiqDecorator < SimpleDelegator`